columnflow/tasks/framework/mixins.py

```diff
@@ -881,7 +881,8 @@ class ProducersMixin(ConfigTask):
 
     producers = law.CSVParameter(
         default=(RESOLVE_DEFAULT,),
-        description="comma-separated names of producers to be applied; empty default",
+        description="comma-separated names of producers to be applied; default: value of the "
+        "'default_producer' config in a 1-tuple",
```

Comment:
I'd remove this last part because this implies to me that I can only pass a single value via my `default_producer` config. Same for the other parameters.
```suggestion
        description="comma-separated names of producers to be applied; default: value of the "
        "'default_producer' config",
```

Reply:
Removed 👍